cli/src/commands/debug/watchman.rs

```diff
@@ -50,6 +50,9 @@ pub fn cmd_debug_watchman(
 
     let mut workspace_command = command.workspace_helper(ui)?;
     let repo = workspace_command.repo().clone();
+    let dummy_watchman_config = WatchmanConfig {
```

Comment:
nit: drop the `dummy_` prefix in case we later add configs to the `WatchmanConfig` (it's possible their value will matter to the debug command so it's not a dummy config anymore then)

Reply:
I replaced it with a comment. I realized I'm only 90% sure the value is irrelevant.